cobald_tests/daemon/core/test_config.py

```diff
@@ -13,6 +15,60 @@
 COBalDLoader.add_constructor(tag="!MockPool", constructor=yaml_constructor(MockPool))
 
 
+class TagTracker:
+    """Helper to track the invocation of YAML !Tags"""
```

Comment:
I don't get what this class is actually supposed to do from within the class definition. Could you maybe explain and extend the docstring to actually document how this is supposed to work/being used?